b/bolt/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
Please copy the file instead of the content, so that formatting is not changed.

Reply:
Replaced the LICENSE file.